packages/Frontend/src/containers/AuthenticateContainer.test.jsx

```diff
@@ -56,11 +45,11 @@ describe('AuthenticateContainer', () => {
   });
 
   describe('signIn', () => {
-    it('should call setState with { error: \'\', loading: true } and a callback', () => {
+    it('should call setState with { error: false, loading: true } and a callback', () => {
       const inst = shallow(<AuthenticateContainer.WrappedComponent {...defaultProps} />).instance();
       inst.setState = jest.fn();
       inst.signIn();
-      expect(inst.setState.mock.calls[0][0]).toEqual({ error: '', loading: true });
+      expect(inst.setState.mock.calls[0][0]).toEqual({ error: false, loading: true });
```

Comment:
This test was changed because with google errors will always be an object, and if we don't have an error we set it to be false.